website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/setup-azure.md

```diff
@@ -49,7 +51,7 @@ Provide your new app access to Azure DevOps:
 2. Remove the **Microsoft Graph / User Read** permission.
 3. Click **Add a permission**.
 4. Select **Azure DevOps**.
-5. Select the **user_impersonation** permission.
+5. Select the **user_impersonation** permission. This is the least scoped and only permission available for Azure DevOps.
```

Comment:
@schottj We should explain "least scoped" a little. It makes me wonder _least scoped compared to what_?
```suggestion
5. Select the **user_impersonation** permission. This permission limits the access to what is absolutely needed to run the Azure DevOps app successfully.
```

Reply:
Maybe we should just say: "This is the only permission available for Azure DevOps."
 
 We had a customer ask if there was any less privileged permission. And the answer is "No, this is the only permission available."